packages/core/src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -90,13 +92,15 @@ export class Autocomplete extends React.PureComponent {
       itemToString,
       label,
       loading,
-      onChange
+      onChange,
+      onInputValueChange
     } = this.props;
 
     return (
       <Downshift
         itemToString={itemToString}
         onChange={onChange}
+        onInputValueChange={onInputValueChange}
```

Comment:
Are we also going to watch out for `onChange` props on children and apply them here if they exist?